lib/api/storage.ts

```diff
@@ -18,11 +20,15 @@ export class ShredderStorage {
   }
 
   static getRaw() {
-    return localStorage.getItem(`${ location.origin }-shredder`);
+    return localStorage.getItem(this.storageKey);
   }
 
   static getSerialized() {
```

Comment:
@ilyagelman Are you sure this is what you meant and not the parse method I added below?